Sofa/framework/Type/src/sofa/type/fwd.h

```diff
@@ -102,6 +105,55 @@ using Quatd = type::Quat<double>;
 using Quatf = type::Quat<float>;
 
 class BoundingBox;
+using BoundingBox3D = BoundingBox;
 class BoundingBox1D;
 class BoundingBox2D;
+
+using FixedArray1i = fixed_array<int, 1>;
+using  FixedArray1I = fixed_array<unsigned int, 1>;
```

Comment:
```suggestion
using FixedArray1I = fixed_array<unsigned int, 1>;
```
My greatest contribution so far